docs/content/modules/products/categories.md

```diff
@@ -0,0 +1,77 @@
+
+A Product Collection is used to group together a list of products, generally for a marketing purpose. For example, you can create a summer collection when summer comes around with a variety of products in them including shorts, t-shirts, and more.
+
+On the other hand, a Product Category is used to provide easy and clear navigation for customers to help them find what they need. For example, you can have the following nested product categories:
+
+- Women
+  - Tops
+    - Jackets
+    - Sweatshirt
+    - T-shirt
+    - etc…
+  - Bottoms
+    - Jeans
+    - Shorts
+    - etc…
+
+This allows customers to directly choose the category that the product they’re looking for belongs to, and gives them an idea of what products your store offers. If you were to use Product Collections here, as they don’t support nesting, all of these categories would have to be separate collections that are not tied together.
+
+---
+
+## ProductCategory Entity Overview
+
+A product category is stored in the database as a `ProductCategory` entity. Some of its important attributes are:
+
+- `id`: The ID of the product category.
+- `name`: The name of the product category.
+- `handle`: A string indicating a slug path of the category. It’s useful when creating a page on your storefront for the category, as the `handle` can be used as the path in the URL.
+- `is_active`: A boolean value indicating the status of the Product Category.
+- `is_internal`: A boolean value indicating the visibility of the Product Category.
+- `mpath`: A string value that keeps the IDs of all the category’s ancestors, delimited by a dot (`.`). This attribute is generated by Typeorm. For example, `pcat_id1.pcat_id2.pcat_id3`.
+
+---
+
+## Relations to other Entities
+
+### Products
+
+Products can be available in more than one category. You can then filter products by a single or list of product categories.
+
+The relation is implemented in the Product entity. You can access the product categories a product is available in by expanding the `categories` relation and using `product.categories`
+
+### ProductCategory
+
+The parent-child hierarchy is represented in the `ProductCategory` entity through the following relations:
+
+- You can access the parent of a category by expanding the `parent_category` relation and accessing `category.parent_category`. You can also access the ID of the parent category by accessing `category.parent_category_id`. If a category doesn’t have a parent, the relation and the ID will be `null`.
+- You can access the children of a category by expanding the `category_children` relation and accessing `category.category_children`. If a category doesn’t have children, the relation will be an empty array.
```

Comment:
Worth it to note here that without `include_descendant_tree=true`, the `category_children` attribute will only return immediate children. with the flag, you will get the entire descendant nodes.

Reply:
Is this flag passed to the endpoints or where exactly?

Reply:
yes, its a flag passed to the endpoint

Reply:
Just checked, its in there. I spelt it wrong here 😅 `include_descendants_tree`